frontend/src/PagesAdmin/AdminLayout/AdminLayout.module.scss

```
@@ -205,3 +217,17 @@ $panel-dark-bg: #333333;
     background-color: $grey-4;
   }
 }
+
+.spacer {
+  flex-grow: 1;
```

Sammen med det andre forslaget

```suggestion
.bottom_items {
  margin-top: auto;
```